Makefile

```diff
@@ -76,6 +76,9 @@ deploy: manifests
 	${KUSTOMIZE} edit remove resource certmanager/certificate.yaml; \
 	else ${KUSTOMIZE} edit add resource certmanager/certificate.yaml; fi;
 	${KUSTOMIZE} build config/default | kubectl apply -f -
+	kubectl wait --for=condition=ready pod -l control-plane=kserve-controller-manager -n kserve --timeout=300s
+	sleep 2
```

Comment:
Is this sleep for applying the runtime resources ?

Reply:
The sleep is for servingruntime validating webhook to be ready. Occasionally,  I am getting webhook errors